Gemfile

```diff
@@ -23,6 +23,7 @@ gem 'jbuilder', '~> 2.7'
 
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
+gem 'bootstrap-sass'
```

Comment:
Normally we lock to only minor version updates when we add gems ... just letting you know for the DocFox codebases ...